content/en/dora_metrics/_index.md

```diff
@@ -279,7 +279,29 @@ Events can be sent both at the start of and after incident resolution. Incident
 
 You can access and visualize your DORA metrics and filter them by team, service, repository, environment, and time period on the [DORA Metrics page][8].
 
-Use the information on this page to identify improvements or regressions for each metric, visualize changes, and compare trends over time. DORA metrics can be exported to dashboards and alerted on using [metric monitors][9].
+Use the information on this page to identify improvements or regressions for each metric, visualize changes, and compare trends over time. DORA metrics can be exported to dashboards or notebooks and be alerted on using [metric monitors][9].
+
+The metrics can also be queried with the [Query timeseries points][10] and [Query timeseries data across multiple products][11] API endpoints.
+
+The metrics provided by DORA Metrics are:
```

Comment:
Yes, can we turn this list into a table with 3 columns: metric name, metric type, and description?